package.json

```diff
@@ -1,13 +1,16 @@
 {
   "license": "Apache-2.0",
   "devDependencies": {
-    "@vuepress/client": "2.0.0-rc.0",
-    "@vuepress/plugin-docsearch": "2.0.0-rc.0",
+    "@vuepress/bundler-webpack": "2.0.0-rc.9",
+    "@vuepress/client": "2.0.0-rc.9",
+    "@vuepress/plugin-docsearch": "2.0.0-rc.19",
     "@vuepress/plugin-google-analytics": "2.0.0-rc.15",
-    "@vuepress/utils": "2.0.0-rc.0",
-    "vue": "^3.3.11",
-    "vuepress": "2.0.0-rc.0",
-    "vuepress-plugin-sitemap2": "2.0.0-rc.4"
+    "@vuepress/plugin-sitemap": "2.0.0-rc.3",
+    "@vuepress/theme-default": "2.0.0-rc.20",
+    "@vuepress/utils": "2.0.0-rc.9",
+    "sass-loader": "^14.1.1",
+    "vue": "^3.4.21",
```

Comment:
We don't explicitly import `vue` and it is already a dependency of `vuepress` so should we be specifying this?

Reply:
I think you're right, I'll remove it!

Reply:
Had to re-add as you can see in the failed actions. Will push once I have updated all the .html links.